daily.php

```diff
@@ -66,7 +66,7 @@
         $rrd_dir = Config::get('rrd_dir');
 
         if (is_numeric($rrd_purge) && $rrd_purge > 0) {
-            $cmd = "find $rrd_dir -type f -mtime +$rrd_purge -print -exec rm -f {} +";
+            $cmd = "find $rrd_dir -name .gitignore -prune -o -type f -mtime +$rrd_purge -print -exec rm -f {} +";
```

Comment:
I think you need to add `\!` before `-name` i.e:

`\! -name`

Reply:
that is what `-prune` does, no `\! -name` required

from man page

       -prune True; if the file is a directory, do not descend into it.  If -depth is given, then -prune has no effect.  Because
              -delete implies -depth, you cannot usefully use -prune and -delete together.  For example, to skip  the  directory
              src/emacs  and all files and directories under it, and print the names of the other files found, do something like
              this:
                  find . -path ./src/emacs -prune -o -print